llvm/include/llvm/ADT/STLExtras.h

```diff
@@ -1881,6 +1881,19 @@ OutputIt move(R &&Range, OutputIt Out) {
   return std::move(adl_begin(Range), adl_end(Range), Out);
 }
 
+/// Provide wrappers to std::includes which take ranges instead of having to
+/// pass begin/end explicitly.
```

Comment:
Maybe it would be worth adding a sentence of explanation of what `includes` does -- it's relatively new (c++17) and not immediately familiar to everyone.